### PR TITLE
Update directory structure of ID parameter control

### PR DESCRIPTION
**Pull request by iago-mendes**
## Proposed changes

This PR addresses issue #6328 by moving the 0th initial data run into the subdirectory `ControlParams00`.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: support/Pipelines/Bbh/InitialData.py [hunk 1]
```diff
@@ -196,6 +198,9 @@ def generate_id(
         )
     if pipeline_dir and not run_dir:
         run_dir = pipeline_dir / "001_InitialData"
+    if control:
+        run_dir = f"{run_dir}/ControlParams00"
```
Comment (nilsvu) [inline support/Pipelines/Bbh/InitialData.py]: Please add an underscore for consistency with pipeline steps and segments directory names: `ControlParams_00`

File: support/Pipelines/Bbh/ControlId.py [hunk 2]
```diff
@@ -166,7 +166,7 @@ def Residual(u):
                 f" Control of BBH Parameters ({iteration}) "
                 "=========================================="
             )
-            control_run_dir = f"{id_run_dir}/ControlParams{iteration:02}"
+            control_run_dir = f"{id_run_dir}/../ControlParams{iteration:02}"
```
Comment (nilsvu) [inline support/Pipelines/Bbh/ControlId.py]: Also add the underscore here

**Review by nilsvu — CHANGES_REQUESTED** [review]

File: support/Pipelines/Bbh/ControlId.py [hunk 3]
```diff
@@ -149,7 +149,7 @@ def control_id(
     radial_expansion_velocity = binary_data["Expansion"]
 
     # File to write control diagnostic data
-    data_file = open(f"{id_run_dir}/ControlParamsData.txt", "w")
+    data_file = open(f"{id_run_dir}/../ControlParamsData.txt", "w")
```
Comment (knelli2) [inline support/Pipelines/Bbh/ControlId.py:152]: @nilsvu Should we avoid relative paths and just pass two arguments? 1) The control params directory, 2) the directory that holds all the control params dirs

Comment (nilsvu) [reply support/Pipelines/Bbh/ControlId.py:152]: I think this is fine, it's confined to the control loop with no public interface that can go wrong

File: support/Pipelines/Bbh/InitialData.py (hunk 1 above)
Comment (iago-mendes) [reply support/Pipelines/Bbh/InitialData.py]: I've also changed to 3 0-padding for the same reason.

**Review by nilsvu — APPROVED** [review]

**Pull request merged**